packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -76,7 +76,7 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 		MessageEncodingContext
 	>;
 
-	private readonly idCompressor: IIdCompressor;
+	public readonly idCompressor: IIdCompressor;
```

Comment:
why was this made public?

Reply:
schematizeFlexTree needs access to it to create the NodeKeyManager. With the way the code ended up being factored though, I can make this protected instead of public though, so I'll do that.

Reply:
Actually in this scope we have access to it via this.runtime.idCompressor, so I guess it can be private.